packages/components/dialog/src/DialogContent.styles.tsx

```diff
@@ -0,0 +1,27 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+export const dialogContentStyles = cva([['fixed', 'z-modal', 'flex', 'flex-col'], ['bg-surface']], {
+  variants: {
+    size: {
+      fullscreen: ['w-screen', 'h-screen'],
```

Comment:
Maybe we should consider using the `dvh` unit instead of `vh` 🤷

Reply:
I'll read about the specifics/differences